src/rngs/thread.rs

```diff
@@ -39,31 +40,43 @@ const THREAD_RNG_RESEED_THRESHOLD: u64 = 1024 * 64;
 
 /// A reference to the thread-local generator
 ///
+/// This type is a reference to a lazily-initialized thread-local generator.
 /// An instance can be obtained via [`thread_rng`] or via `ThreadRng::default()`.
 /// This handle is safe to use everywhere (including thread-local destructors),
 /// though it is recommended not to use inside a fork handler.
 /// The handle cannot be passed between threads (is not `Send` or `Sync`).
 ///
-/// `ThreadRng` uses the same PRNG as [`StdRng`] for security and performance
-/// and is automatically seeded from [`OsRng`].
+/// `ThreadRng` uses the same CSPRNG as [`StdRng`], ChaCha12. As with
+/// [`StdRng`], the algorithm may be changed, subject to reasonable expectations
+/// of security and performance.
 ///
-/// Unlike `StdRng`, `ThreadRng` uses the  [`ReseedingRng`] wrapper to reseed
-/// the PRNG from fresh entropy every 64 kiB of random data as well as after a
-/// fork on Unix (though not quite immediately; see documentation of
-/// [`ReseedingRng`]).
-/// Note that the reseeding is done as an extra precaution against side-channel
-/// attacks and mis-use (e.g. if somehow weak entropy were supplied initially).
-/// The PRNG algorithms used are assumed to be secure.
+/// `ThreadRng` is automatically seeded from [`OsRng`] with periodic reseeding
+/// (every 64 kiB, as well as "soon" after a fork on Unix — see [`ReseedingRng`]
+/// documentation for details).
+///
+/// Security must be considered relative to a thread model and validation
```

Comment:
Typo: threat -> threat

Reply:
Thanks, but this is merged already. Can you make a PR? (Should be possible through the GitHub web UI.)